src/platformdirs/android.py

```diff
@@ -56,6 +56,25 @@ def user_log_dir(self) -> str:
             path = os.path.join(path, "log")
         return path
 
+    @property
+    def user_documents_dir(self) -> str:
+        """
+        :return: documents directory tied to the user e.g. ``/data/media/0/Documents``
+        """
+        # Get directories with pyjnius
+        try:
+            from jnius import autoclass  # noqa: SC200
+
+            Environment = autoclass("android.os.Environment")
+            data_dir: str = Environment.getDataDirectory().getAbsolutePath()
+            documents_dir_name: str = Environment.DIRECTORY_DOCUMENTS
+
+            documents_dir = os.path.join(data_dir, "media", "0", documents_dir_name)
```

Comment:
It will be better to make this (define folder in `/storage/emulated/0`) with `Context`, because some methods of `Environment` will be deprecated since 29 API level, e.g.

```python
Context = autoclass('android.content.Context')

documents_dir = Context.getExternalFilesDir(Environment.DIRECTORY_DOCUMENTS).getAbsolutePath()
```
Of course it will return APP directory NOT USER like `/storage/emulated/0/Android/data/com.example/files/Documents`

Reply:
I'm having trouble understanding what you're suggesting here. Doesn't this need to be the user documents folder? It sounds like you're asking me to do it with `Context` and with `/storage/emulated/0` somehow, but not with `Environment`. Could you rephrase your comment?

Reply:
I am afraid of using
```python
Environment.getExternalStorageDirectory(Environment.DERICTORY_DOCUMENTS)
``` which returns `/storage/emulated/0` and will be depricatedyou, so suggest to do it with `Context`. You did it via `os.path.join` what is the place to be

Reply:
Yes